tests/func/test_get.py

```diff
@@ -87,3 +88,33 @@ def test_get_to_dir(dname, repo_dir, erepo):
 
     assert os.path.isdir(dname)
     assert filecmp.cmp(repo_dir.FOO, dst, shallow=False)
+
+
+@pytest.fixture
+def erepo_no_dvc_master(git_erepo):
+    dvc_branch = "dvc_test"
+    git_erepo.git.git.checkout("master", b=dvc_branch)
+    git_erepo.dvc_branch = dvc_branch
+
+    dvc_repo = Repo.init(git_erepo._root_dir)
+    stage, = dvc_repo.add([git_erepo.FOO])
+    dvc_repo.scm.add([".dvc", stage.relpath])
+
+    rconfig = RemoteConfig(dvc_repo.config)
+    rconfig.add("upstream", dvc_repo.cache.local.cache_dir, default=True)
+    dvc_repo.scm.add([dvc_repo.config.config_file])
```

Comment:
Can be removed after #2780

Reply:
We only use it once. Why not just inline it into the test?